packages/mds-agency/request-handlers.ts

```diff
@@ -31,11 +31,23 @@ import {
   readPayload,
   computeCompositeVehicleData
 } from './utils'
+import {
+  AgencyRegisterVehicleResponse,
+  AgencyGetVehicleByIdResponse,
+  AgencyGetVehiclesByProviderResponse,
+  AgencyUpdateVehicleResponse,
+  AgencySubmitVehicleEventResponse,
+  AgencySubmitVehicleTelemetryResponse,
+  AgencyRegisterStopResponse,
+  AgencyReadStopsResponse,
+  AgencyReadStopResponse
+} from './types'
 
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 stream.initialize()
+const agencyServerError = { error: 'server_error', error_description: 'Unknown server error' }
```

Comment:
It's not _in_ the spec, but nothing's currently specified for 500s and this at least follows the format.